### PR TITLE
feat: add db interaction in system tests

### PR DESCRIPTION
**Pull request by stepanLav**
That PR adds logic to interact with browser context inside system tests in order to add any data directly into database. That approach provide a way to add any test data in order to prepare the application to test.

Also that include:
- Updates for `Run system tests` workflow, now it will be run:
-- daily, at the 9.00 with  `@fee-test` tag
-- after each merge to dev with `@regress` tag
-- manually without any tags (whole run)
-- when change anything inside tests directory (whole run)
- Added notification about test run in spektr_notifications
- Added test to check default settings on assets tab - 'Default settings for assets page'
- Added test to check fee calculation for substrate based chains - 'Load Transfer fee as Substrate Polkadot Vault'
- Added test to check fee calculation for eth based chain - 'Load Transfer fee as ethereum_based polkadot Vault wallet'
- Was updated docker image node version

### CODE REVIEW

File: tests/system/tests/onboarding/watch.only.onboarding.system.test.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { test, expect } from '@playwright/test';
+
+import { BaseLoginPage } from '../../pages/loginPage/BaseLoginPage';
+import { LoginPageElements } from '../../pages/_elements/LoginPageElements';
+import { baseTestConfig } from '../../BaseTestConfig';
+
+test.describe(
+  'Watch only wallet onboarding',
+  {
+    tag: '@regress',
```
Comment (stepanLav) [inline tests/system/tests/onboarding/watch.only.onboarding.system.test.ts:11]: regress tag is used for runs after merge in dev branch

File: scripts/updateTestData.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+
+const updateChainsList = () => {
```
Comment (stepanLav) [inline scripts/updateTestData.js:4]: This is a preparation script that will run for each test run and update the test data for the available chains to have it in a TypeScript object, making it possible to specify this data as an iteration object in tests.

File: tests/system/tests/onboarding/vault.onboarding.system.test.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { chromium, Browser, Page, BrowserContext } from 'playwright';
+import { test, expect } from '@playwright/test';
+
+import { BaseLoginPage } from '../../pages/loginPage/BaseLoginPage';
+import { LoginPageElements } from '../../pages/_elements/LoginPageElements';
+
+test.describe(
+  'Polkadot Vault onboarding',
+  {
+    tag: '@regress',
```
Comment (stepanLav) [inline tests/system/tests/onboarding/vault.onboarding.system.test.ts:10]: regress tag is used for runs after merge in dev branch

File: tests/system/tests/assets/load.fee.test.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+import { BaseLoginPage } from '../../pages/loginPage/BaseLoginPage';
+import { LoginPageElements } from '../../pages/_elements/LoginPageElements';
+import { substrateChains } from '../../data/chains/chainsList';
+import { VaultAssetsPage } from '../../pages/assetsPage/VaultAssetsPage';
+
+test.describe(
+  'Load Transfer fee as Substrate Polkadot Vault',
+  {
+    tag: '@fee-test',
```
Comment (stepanLav) [inline tests/system/tests/assets/load.fee.test.ts:12]: run `@fee-test` tests daily, in order to catch problems with runtime upgrade

File: tests/system/tests/assets/load.eth.fee.test.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+import { BaseLoginPage } from '../../pages/loginPage/BaseLoginPage';
+import { LoginPageElements } from '../../pages/_elements/LoginPageElements';
+import { ethChains } from '../../data/chains/chainsList';
+import { VaultAssetsPage } from '../../pages/assetsPage/VaultAssetsPage';
+
+test.describe(
+  'Load Transfer fee as ethereum_based polkadot Vault wallet',
+  {
+    tag: '@fee-test',
```
Comment (stepanLav) [inline tests/system/tests/assets/load.eth.fee.test.ts:12]: run `@fee-test` tests daily, in order to catch problems with runtime upgrade

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**